src/Features/CSharp/Portable/CodeFixes/HideBase/HideBaseCodeFixProvider.AddNewKeywordAction.cs

```diff
@@ -51,7 +53,8 @@ private SyntaxNode GetNewNode(Document document, SyntaxNode node, CancellationTo
                 var fieldDeclaration = node as FieldDeclarationSyntax;
                 if (fieldDeclaration != null)
                 {
-                    newNode = fieldDeclaration.AddModifiers(SyntaxFactory.Token(SyntaxKind.NewKeyword));
+                    var generator = _document.GetLanguageService<SyntaxGenerator>();
+                    newNode = generator.WithModifiers(fieldDeclaration, generator.GetModifiers(fieldDeclaration).WithIsNew(true));
```

Comment:
I thnk this can be even simpler.  This entire method can just be:

```return generator.WithModifiers(fieldDeclaration, generator.GetModifiers(fieldDeclaration).WithIsNew(true));```

We don't need any of the rest of the code in this method.